scipy/linalg/_decomp_qr.py

```diff
@@ -51,8 +51,9 @@ def qr(a, overwrite_a=False, lwork=None, mode='full', pivoting=False,
     pivoting : bool, optional
         Whether or not factorization should include pivoting for rank-revealing
         qr decomposition. If pivoting, compute the decomposition
-        ``A P = Q R`` as above, but where P is chosen such that the diagonal
-        of R is non-increasing.
+        :math:`A_P = Q R` as above, but where :math:`A_P` is the matrix A with 
```

Comment:
`A_P` is confusing here. Simple NumPy code block is better for this `A[:, P] = Q @ R` for readability.

Reply:
thank you, how about now? 
a2c99a2